### PR TITLE
Add memcache extension layer

### PR DESCRIPTION
**Pull request by jenschude**

### CODE REVIEW

**Comment by jenschude** [comment]
I didn't had the time to check if the extension is working. Maybe next year if you're not faster ;)

**Comment by mnapoli** [comment]
Nice! It's worth making sure it works before merging though (whoever finds the time ^^).

File: layers/memcached/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+WORKDIR  ${LIBMEMCACHED_BUILD_DIR}/libmemcached-1.0.18/
+
+# Install
+RUN set -xe; \
+    patch -d . -p0 < libmemcached-build.patch && \
+    ./configure --prefix=${INSTALL_DIR} && \
+    make && \
+    make install
+
+RUN pecl install memcache
```
Comment (Nyholm) [inline layers/memcached/Dockerfile]: Are you sure? 
Dont you mean memcached?

Comment (Nyholm) [reply layers/memcached/Dockerfile]: Or we maybe should add both?

Comment (jenschude) [reply layers/memcached/Dockerfile]: Haha funny. Yeah good question. memcache is the extension for memcached.

memcached for use with libmemcached

Comment (Nyholm) [reply layers/memcached/Dockerfile]: I add both PHP extensions with this docker image.. Im currently building and testing.

**Comment by Nyholm** [comment]
Screenshot 2019-12-30 at 17 15 26

Screenshot 2019-12-30 at 17 16 20


It is working (with some minor adjustments.

**Comment by Nyholm** [comment]
I pushed my changes to your branch.

**Comment by Nyholm** [comment]
Thank you for your work!

**Comment by Nyholm** [comment]
I have published the layer now. 

You can try install dev-master of the php package and try out your extension if you want to.

**Pull request merged**